python/dgl/distributed/rpc_server.py

```diff
@@ -108,7 +108,10 @@ def start_server(server_id, ip_config, num_servers, num_clients, server_state, \
                     register_res = rpc.ClientRegisterResponse(client_id)
                     rpc.send_response(client_id, register_res, group_id)
         # receive incomming client requests
-        req, client_id, group_id = rpc.recv_request()
+        timeout = 5 * 1000  # in milliseconds
+        req, client_id, group_id = rpc.recv_request(timeout)
+        if req is None:
+            continue
```

Comment:
Why continue instead of popping errors?

Reply:
server needs to always waiting for any coming requests